src/js/oCrossword.js

```diff
@@ -462,7 +472,7 @@ OCrossword.prototype.assemble = function assemble() {
 
 		}.bind(this);
 
-		this.onResize = debounce(onResize, 100);
+		if(!isAndroid())	this.onResize = debounce(onResize, 100);
```

Comment:
Can we wrap `if` statements in `{ }`, please? If we ever refactor this, these sort of statements can make the job a nightmare.